q2_longitudinal/plugin_setup.py

```diff
@@ -171,14 +171,14 @@
     description=(
         'Performs paired difference testing between samples from each '
         'subject. Sample pairs may represent a typical intervention study '
-        '(e.g., samples collected pre- and post-treatment), paired samples from '
-        'two different timepoints (e.g., in a longitudinal study design), or '
-        'identical samples receiving different treatments. This action tests '
-        'whether the change in a numeric metadata value "metric" differs from '
-        'zero and differs between groups (e.g., groups of subjects receiving '
-        'different treatments), and produces boxplots of paired difference '
-        'distributions for each group. Note that "metric" can be derived from '
-        'a feature table or metadata.')
+        '(e.g., samples collected pre- and post-treatment), paired samples '
+        'from two different timepoints (e.g., in a longitudinal study design),'
+        ' or identical samples receiving different treatments. This action '
```

Comment:
could you standardize this here and below so that the spaces between words appear at the end of the lines instead of the beginning? Thanks!

Reply:
Yep.